yarn.lock

```diff
@@ -159,6 +159,13 @@
   dependencies:
     "@turf/helpers" "^5.1.5"
 
+"@turf/kinks@5.1.x", "@turf/kinks@^5.1.5":
+  version "5.1.5"
```

Comment:
Is there any reason why this global file should be change?

Reply:
Sorry It is not intentional.
Maybe I did it by mistake.
I will revert it in next commit.